### DISCUSSION TITLE
Git for Windows v2.48.0-rc1.windows.1

### DISCUSSION THREAD

**Discussion by gitforwindowshelper** [git-for-windows/git#5339] [Announcements]
Changes since Git for Windows v2.47.1 (November 25th 2024)

### New Features

* Comes with [Git v2.48.0-rc1](https://github.com/git/git/blob/v2.48.0-rc1/Documentation/RelNotes/2.48.0.txt).
* Comes with [cURL v8.11.1](https://curl.se/changes.html#8_11_1).
* Comes with [MinTTY v3.7.7](https://github.com/mintty/mintty/releases/tag/3.7.7).
* ~~Comes with the MSYS2 runtime (Git for Windows flavor) based on [Cygwin v3.5.5](https://inbox.sourceware.org/cygwin-announce/20241220181956.2204526-1-corinna-cygwin@cygwin.com).~~ Git for Windows 2.48.0-rc1.windows.1 was supposed to come with the MSYS2 runtime (Git for Windows flavor) based on Cygwin v3.5.5. Due to random hangs that couldn't be fixed in time, this has been reverted.

### Bug Fixes

* The installer now correctly blocks the installation on Windows 7 and Windows 8 as these versions of Windows are no longer supported since Git for Windows v2.47.0

Filename | SHA-256
-------- | -------
Git-2.48.0-rc1-64-bit.exe | 205d78a134419d8c4cae8a8287f3cd43df1577e3548f9aa298023c95bbafec2b
Git-2.48.0-rc1-32-bit.exe | a0333a4a5f4f91aa91587efc0b2171940059445c39dde54284f320eec0b3975b
PortableGit-2.48.0-rc1-64-bit.7z.exe | 4547c8db412d7a1b840074701e16354458181a54b7eff838eec3ba3023e079ea
PortableGit-2.48.0-rc1-32-bit.7z.exe | 6fd50d8726793006d1961fd9d03c1cfc94a4994f78bacfd835c1ab58f1d5b56c
MinGit-2.48.0-rc1-64-bit.zip | 139255b6f7cdd14c696da58e17bd3284cd7f0975fb7c45f20f8cfca8115acdd2
MinGit-2.48.0-rc1-32-bit.zip | ea5eb4332d51455dd5850c9a4a265d158b8a7bd8d9d280a0bc7c485d4e77dcc5
MinGit-2.48.0-rc1-busybox-64-bit.zip | 769206ce840d18ad1d6185ba2fb56c8593aebfbb9a26aae9850dbd35b35a2a6b
MinGit-2.48.0-rc1-busybox-32-bit.zip | 92a16a3b491ba4c6c6f68d30ba29360bb75db014a834c8947cd543d427be784e
Git-2.48.0-rc1-64-bit.tar.bz2 | 399e604024fc3c3c476aea8d817c39a31d509c82741379a5c7cd49b50ba75b42
Git-2.48.0-rc1-32-bit.tar.bz2 | e055a18ac811ac77a6372eefe1833dacc91a0add9c8cd2d971b22a226a30f0b4
Git-2.48.0-rc1-arm64.tar.bz2 | f37ce1adbd91c195cf333f8713ae7964dbf169b1de5882353e03b727f4ec8e79
Git-2.48.0-rc1-arm64.exe | dcbf2a6ff870d498ae8aeb53c0d173a393ce6d7218bd6daa7ce2c416b78b743b
MinGit-2.48.0-rc1-arm64.zip | 8f56c87ceb5c5b41a5690554fe091b8b9aa08748bb20c39e933b0a2c8546ef01
PortableGit-2.48.0-rc1-arm64.7z.exe | 594ab37e5e378efb30a97796fa38f3a5d833f82da223d4056e596f311270fe21

<hr /><em>This discussion was created from the release <a href='https://github.com/git-for-windows/git/releases/tag/v2.48.0-rc1.windows.1'>Git for Windows v2.48.0-rc1.windows.1</a>.</em>

**Comment by rimrul** [comment]
Note: The release notes that come bundled with Git for Windows 2.48.0-rc1.windows.1 and the announcement mails falsely claim that this version comes with an updated MSYS2 runtime. I've updated the release notes in this discussion and the release description to accurately reflect that this change has been reverted.
